INTEGRATING.md

```diff
@@ -1,34 +1,103 @@
-## Integrating Loki
+## Integrating Loki Charmed Operator
 
-Loki integrates with the following:
+### Overview
 
+This Loki Charmed Operator provides two libraries so you can use them to integrate it
+with other charms.
 
-1. Any charm that supports the `loki_push_api` interface.
-You can integrate it using the Loki Charm library. The documentation of
-the charm library, available through [Charmhub](https://charmhub.io/loki-k8s)
-provides further details. The information
-exchanged through the `loki_push` interface can be broken down
-into two parts
+- [`loki_push_api`](https://charmhub.io/loki-k8s/libraries/loki_push_api)
+- [`log_proxy`](https://charmhub.io/loki-k8s/libraries/log_proxy)
 
 
-- Loki charm provides an [endpoint URL](https://grafana.com/docs/loki/latest/api/#post-lokiapiv1push)
-to receive log from [Loki clients](https://grafana.com/docs/loki/latest/clients/) that
-relates with this charm.
+The library `loki_push_api` provides
+[endpoint URL](https://grafana.com/docs/loki/latest/api/#post-lokiapiv1push)
+to receive log from [Loki clients](https://grafana.com/docs/loki/latest/clients/) that relates
+with this charm.
```

Comment:
This part is a little confusing: it mentions charms but links to the underlying software.

Reply:
Definitely we need a better wording here.